ivy/functional/backends/jax/experimental/linear_algebra.py

```diff
@@ -2,6 +2,8 @@
 from typing import Optional, Tuple, Sequence, Union
 import jax.numpy as jnp
 import jax.scipy.linalg as jla
+
+from ivy import with_supported_dtypes
```

Comment:
let's probably do `from ivy.func_wrapper import with_supported_dtypes` following the general convention and remove the additional blank line